src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -109,7 +109,9 @@ public enum ErrorCode {
   VECTORIZE_SERVICE_TYPE_UNSUPPORTED("Vectorize service type unsupported : "),
 
   VECTORIZE_SERVICE_TYPE_UNAVAILABLE("Vectorize service unavailable : "),
-  VECTORIZE_USAGE_ERROR("Vectorize search can't be used with other sort clause");
+  VECTORIZE_USAGE_ERROR("Vectorize search can't be used with other sort clause"),
+
+  VECTORIZECONFIG_CHECK_FAIL("VectorizeConfig check null fail");
```

Comment:
This is a pretty bad failure message to give to user -- what can they do with it? It refers to some internal details that caller has no access to or knowledge of.

We may need to instead indicate some sort of "internal processing failure", related to bad configuration for vectorization?